lib/vquic/curl_quiche.c

```diff
@@ -344,6 +344,8 @@ static CURLcode write_resp_raw(struct Curl_cfilter *cf,
   ssize_t nwritten;
 
   (void)cf;
+  if(!stream)
```

Comment:
Compiler complaines about: `quiche_stream_iter *qiter = quiche_conn_writable(ctx->qconn);`

May this will fix it?:

```
if(!stream) {
   return FALSE;
}
```

Reply:
Thanks! Just pused a fix.